### ISSUE TITLE
[js_basics] Add instructions on how to add scripts on page

### ISSUE THREAD

**Issue by sudodoki** [kottans/js-slides#11]
1) Inclusion of js on page: `<script src='...'></script>` vs `<script>/*code*/</script>`
2) Unblock UI: async/defer/dynamic script create
3) Position: head vs end of the body

**Comment by nicohq** [comment]
->2. It might be helpful - [article](http://www.growingwiththeweb.com/2014/02/async-vs-defer-attributes.html?utm_content=buffer78108&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer). Large [img](https://pbs.twimg.com/media/BySaPN-IMAAJVnz.png:large)

**Comment by sudodoki** [comment]
:+1: looks nice

**Comment by sudodoki** [comment]
`wontfix` info is there http://kottans.org/js-slides/js_dom/#/5.
